executor/analyze.go

```diff
@@ -980,26 +1034,30 @@ func (e *AnalyzeColumnsExec) buildSamplingStats(
 	e.samplingMergeWg = &util.WaitGroupWrapper{}
 	e.samplingMergeWg.Add(statsConcurrency)
 	for i := 0; i < statsConcurrency; i++ {
-		go e.subMergeWorker(mergeResultCh, mergeTaskCh, l, i == 0)
+		go e.subMergeWorker(mergeResultCh, mergeTaskCh, l, i, e.memTracker)
```

Comment:
we don't need to pass `e.memTracker` here as it is already hold by `e`

Reply:
Updated